modules/gpu_node_pool/gpu_node.tpl

```diff
@@ -3,51 +3,44 @@
 export HOME=/root
 
 function nvidia_configure() {
- sudo pkill -SIGHUP dockerd && \
  distribution=$(. /etc/os-release;echo $ID$VERSION_ID) ; \
  curl -s -L https://nvidia.github.io/nvidia-docker/gpgkey | sudo apt-key add - ; \
  curl -s -L https://nvidia.github.io/nvidia-docker/$distribution/nvidia-docker.list | sudo tee /etc/apt/sources.list.d/nvidia-docker.list ; \
- sudo apt-get update && sudo apt-get install -y nvidia-docker2 nvidia-cuda-toolkit ; \
- sudo systemctl restart docker
+ sudo apt-get update && sudo apt-get install -y nvidia-container-runtime nvidia-cuda-toolkit
 }
 
 function nvidia_drivers() {
   sudo sed -i 's/^#root/root/' /etc/nvidia-container-runtime/config.toml && \
   sudo tee /etc/modules-load.d/ipmi.conf <<< "ipmi_msghandler" \
   && sudo tee /etc/modprobe.d/blacklist-nouveau.conf <<< "blacklist nouveau" \
   && sudo tee -a /etc/modprobe.d/blacklist-nouveau.conf <<< "options nouveau modeset=0" && \
-  sudo update-initramfs -u && \
-  sudo docker run --name nvidia-driver -d --privileged --pid=host \
-  -v /run/nvidia:/run/nvidia:shared \
-  -v /var/log:/var/log \
-  --restart=unless-stopped \
-  nvidia/driver:450.80.02-ubuntu18.04
+  sudo update-initramfs -u
+  sudo ctr i pull docker.io/nvidia/driver:450.80.02-ubuntu18.04
+  sudo ctr run --rm --privileged --with-ns pid:/proc/1/ns/pid --net-host --detach \
+  --mount type=bind,src=/run/nvidia,dest=/run/nvidia,options=shared \
```

Comment:
The GPU configuration may need more testing. The /run/nvidia path did not exist when I attempted this.  The removal of `--restart=unless-stopped` is also worth consideration.  What is the containerd equivalent?